### PR TITLE
chore: Update changelog for auth fixes

### PR DESCRIPTION
**Pull request by royjit**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Bug fixes
+
+- **Amazon Cognito**
+  - Fixed an issue where the aws credentials are not cleared during token expiry (See [PR #3237](https://github.com/aws-amplify/aws-sdk-ios/pull/3237))
+  - Fixed an issue where identity id is not updated immediately after signin (See [Issue #2922](https://github.com/aws-amplify/aws-sdk-ios/issues/2922), [PR #3236](https://github.com/aws-amplify/aws-sdk-ios/pull/3236))
```
Comment (lawmicha) [inline CHANGELOG.md:9]: Could we just re-use the PR name?
```suggestion
  - Update id token immediately after signIn (See [Issue #2922](https://github.com/aws-amplify/aws-sdk-ios/issues/2922), [PR #3236](https://github.com/aws-amplify/aws-sdk-ios/pull/3236))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Bug fixes
+
+- **Amazon Cognito**
+  - Fixed an issue where the aws credentials are not cleared during token expiry (See [PR #3237](https://github.com/aws-amplify/aws-sdk-ios/pull/3237))
```
Comment (lawmicha) [inline CHANGELOG.md:8]:
```suggestion
  - Remove cached AWS Credentials when token expire (See [PR #3237](https://github.com/aws-amplify/aws-sdk-ios/pull/3237))
```

**Review by lawmicha — APPROVED** [review]
some suggestions

File: CHANGELOG.md (hunk 2 above)
Comment (royjit) [reply CHANGELOG.md]: Updated

File: CHANGELOG.md (hunk 1 above)
Comment (royjit) [reply CHANGELOG.md]: Updated

**Pull request merged**